doc/release-notes.md

```diff
@@ -57,6 +57,11 @@ Notable changes
 Example item
 ------------
 
+Tools and Utilities
+-------------------
+
+- Allow logging with severity level and display it with the category name. (#24464)
```

Comment:
As it is not clear that this pull will be merged quickly, maybe move the 5 lines to a separate file, per the instructions in https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#release-notes ?

Reply:
Done 👍🏻